cirq/google/calibration/phased_fsim.py

```diff
@@ -198,9 +198,11 @@ def override_by(self, other: 'PhasedFSimCharacterization') -> 'PhasedFSimCharact
         return other.merge_with(self)
 
 
-SQRT_ISWAP_PARAMETERS = PhasedFSimCharacterization(
+SQRT_ISWAP_INV_PARAMETERS = PhasedFSimCharacterization(
     theta=np.pi / 4, zeta=0.0, chi=0.0, gamma=0.0, phi=0.0
 )
+# Pending deprecation in 0.11.
+SQRT_ISWAP_PARAMETERS = SQRT_ISWAP_INV_PARAMETERS
```

Comment:
Please add this test to `phased_fsim_test` to check that the deprecated variable still works:
```
def test_sqrt_iswap_parameters_deprecation():
    with pytest.warns(DeprecationWarning):
        assert SQRT_ISWAP_PARAMETERS == SQRT_ISWAP_INV_PARAMETERS
```

Reply:
I added the tests but it's failing because it complains I'm using a deprecated parameter in Cirq. Do I do something wrong here?

Reply:
I think the issue is due to importing the variable by name outside of the `pytest.warns` block. Our CI tests treat any use of deprecated fields as failures unless they're captured by one of these blocks.

Reply:
Sorry, I misunderstood our deprecation-testing behavior. In place of the current test, you'll want to use `assert_deprecated` instead:
https://github.com/quantumlib/Cirq/blob/69daec18bb2d2cdd7ca8ffc6158b2f1658c27dd5/cirq/testing/deprecation.py#L23

Reply:
I think that I fixed whatever I could easily fix. Once I modified `cirq.google.__init_-`, the following error started to appear: `ValueError: cirq.google.__spec__ is None`. Would you know where it comes from?

Reply:
I'm not familiar with this error, but it looks like it's related to the JSON modularization changes.

@balopat, could you take a look at this?